docs/writing-policies/go/03-policy-settings.md

```diff
@@ -11,160 +11,105 @@ We will do that by adding this code inside of the `settings.go` file:
 
 ```go
 import (
-	"github.com/deckarep/golang-set"
-	"github.com/kubewarden/gjson"
+	mapset "github.com/deckarep/golang-set"
 	kubewarden "github.com/kubewarden/policy-sdk-go"
+	kubewarden_protocol "github.com/kubewarden/policy-sdk-go/protocol"
+	"github.com/mailru/easyjson"
 
 	"fmt"
 	"regexp"
 )
 
 type Settings struct {
```

Comment:
shouldn't we defined the Settings in [type.go](https://github.com/kubewarden/go-policy-template/blob/main/types.go#L21) ? 
Adding Settings here gives me a duplicated struct error

Reply:
ah I saw the next step is removing the Settings in type.go . Will do all the steps